Propagation-Scripts/windows_service/script.ps1

```diff
@@ -25,12 +25,15 @@ param(
     [string]$ServiceName,
 
     [Parameter()]
-    [ValidateSet('yes','')]
+    [ValidateSet('yes', '')]
     [string]$RestartService
 )
 
-# Output the script parameters and the current user running the script
-Write-Output -InputObject "Running script with parameters: $($PSBoundParameters | Out-String) as [$(whoami)]"
+function Write-DvlsHost($Message) {
+    Write-Output $Message
+}
+
+Write-DvlsHost "Starting script execution with parameters: $($PSBoundParameters | Out-String) as [$(whoami)]"
```

Comment:
Is it dangerous to write in the output the password receive as parameter ?

Reply:
No, as it will just return an encrypted string.